process-engine/bpmn-studio
# ✨ Allow for customized event payloads

**Changes:**

Apply a fix for the ProcessEngineRuntime, which prevented the use of the `inputValues` property for events that use signals and messages.

PR: #1440

## How can others test the changes?

- Create a ProcessModel that has a StartEvent that waits for a message or signal.
- Create a ProcessModel that uses  an `EndEvent` or `IntermediateThrowEvent` that sends the expected mesage or a signal.
- Attach an `inputValues` property to the triggering event that performs some kind of token mapping.
- Run the ProcessModel that uses the triggering event.
- When the ProcessModel with the StartEvent gets run, see that it received the correct payload.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
